department-of-veterans-affairs/va.gov-cms
# VACMS-1593: Install and enable fieldhelptext module.

## Description

See #1593. 

## Testing done

As admin go to /admin/structure/fieldhelptext

Edit help text for all fields on a content type:
* Click event (/admin/structure/fieldhelptext/by-bundle/node/event)
* Update help text for all event fields
* Go to /node/add/event and see your changes. 

Edit one field across all content types
* Go back to  /admin/structure/fieldhelptext
* In the Edit by Field section, click field_intro_text
* Edit the field 
* Change the field label and Description
* Open a bunch of forms that contain this field and see that your changes are there.
